File: 01-super-basic-questions/12-sum-array-eles/solution.py
arrSize = int(input("Enter the size of the array: "))
arr = [int(input("Enter ele: ")) for _ in range(arrSize)]
print(arr, end="\n")

sum = 0

for i in range(arrSize):
    sum += arr[i]
    
print("Sum of array's elements: "+str(sum))

File: 01-super-basic-questions/12-sum-array-eles/solution.go
package main

import "fmt"

func main() {
	var arrSize int
	fmt.Println("Enter the size of the array: ")
	fmt.Scan(&arrSize)

	arr := make([]int, arrSize)

	for i := 0; i < arrSize; i++ {
		fmt.Scan(&arr[i])
	}

	sum := 0

	for i := 0; i < arrSize; i++ {
		sum += arr[i]
	}

	fmt.Println("Sum of arr elements: ", sum)

}
